src/js/blue-button/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+
+import form from './form';
+
+export default combineReducers({
+  form,
```

Comment:
I prefer to not have the trailing comma, but not a strong preference. If this is the pattern you've been using for the other rx products, consistency is better than my preference 😄